### PR TITLE
Update CARMA-Streets plugin to broadcast SPaT include MovementEventList

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description
CARMA-Platform vehicles need information about future planned phase state for each signal group to plan trajectory through signalized intersection. This information will be retrieved from the following fields of the J2735 define SPaT message.

## Related Issue
https://github.com/usdot-fhwa-OPS/V2X-Hub/issues/407

## Motivation and Context
UC3

## How Has This Been Tested?
Unit tests

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/v2i-hub/CARMAStreetsPlugin/manifest.json [hunk 1]
```diff
@@ -46,6 +46,11 @@
             "default": "v2xhub_scheduling_plan_sub",
             "description": "Apache Kafka topic plugin will transmit message to."
         },
+        {
+            "key": "subscribeToSpatTopic",
+            "default": "spat_json",
```
Comment (paulbourelly999) [inline src/v2i-hub/CARMAStreetsPlugin/manifest.json:51]: @dan-du-car Saeid created a topic name as well which he called "modified_spat". I don't care which one we use but they should be the same.

File: src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.cpp [hunk 2]
```diff
@@ -77,39 +79,56 @@ void CARMAStreetsPlugin::UpdateConfigSettings() {
...
-	_topic = RdKafka::Topic::create(kafka_consumer,_subscribeToSchedulingPlanTopic,tconf,error_string);
-	if(!_topic)
+	_scheduing_plan_topic = RdKafka::Topic::create(_scheduing_plan_kafka_consumer,_subscribeToSchedulingPlanTopic,tconf_sp,error_string);
+	if(!_scheduing_plan_topic)
+	{
+		PLOG(logERROR) << "RDKafka create scheduing plan topic failed:" << error_string;
+		return ;
+	}
+
+	_spat_topic = RdKafka::Topic::create(_spat_kafka_consumer,_subscribeToSpatTopic,tconf_spat,error_string);
```
Comment (paulbourelly999) [inline src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.cpp:120]: do we need to create the topic here or will it already be created by docker-compose setup.

**Review by paulbourelly999 — CHANGES_REQUESTED** [review]
Looks good to me. Once you update the topic name I am good to merge.

File: src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.cpp (hunk 2 above)
Comment (dan-du-car) [reply src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.cpp:120]: This is for the consumer config, it is not the same as create kafka topic and register to kafka broker

**Review by paulbourelly999 — APPROVED** [review]

**Pull request merged**